lib/rules/aria-hidden-body.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "aria-hidden-body",
+  "selector": "body",
+  "tags": [
+    "cat.aria",
+    "wcag2a",
+    "wcag411",
+    "wcag412"
+  ],
+  "metadata": {
+    "description": "Ensures aria-hidden is not present on the document body.",
+    "help": "Document body should not include the aria-hidden attribute."
```

Comment:
Perhaps describe the problem this causes? e.g. using `aria-hidden="true"` makes the entire page unusable by screen readers?